diff --git a/Makefile b/Makefile
index 0f7ea29..73c6134 100644
--- a/Makefile
+++ b/Makefile
@@ -5,14 +5,12 @@
 ## Default compilation flags.
 ## Override with:
 ##   make CXXFLAGS=XXXXX
-CXXFLAGS= -O3 -g -D__STDC_LIMIT_MACROS -D_FILE_OFFSET_BITS=64 -std=c++0x -DMACOSX -pthread #-pedantic -Wunreachable-code -Weverything
+CXXFLAGS=$(CPPFLAGS) -O3 -g -D__STDC_LIMIT_MACROS -D_FILE_OFFSET_BITS=64 -std=c++0x -DMACOSX -pthread #-pedantic -Wunreachable-code -Weverything
 
 ## To create a static distribution file, run:
 ##   make static-dist
 ifeq ($(STATIC),1)
-LDFLAGS=-static
-else
-LDFLAGS=
+LDFLAGS += -static
 endif
 
 ## Source code files, add new files to this list
@@ -43,7 +41,7 @@ all: HipSTR DenovoFinder test/fast_ops_test test/haplotype_test test/read_vcf_al
 static-dist:
 	rm -f HipSTR
 	$(MAKE) STATIC=1
-	( VER="$$(git describe --abbrev=7 --dirty --always --tags)" ;\
+	( VER="$$(grep 'version:' _config.yml | awk -F': *' '{print $2}')" ;\
 	  DST="HipSTR-$${VER}-static-$$(uname -s)-$$(uname -m)" ; \
 	  mkdir "$${DST}" && \
             mkdir "$${DST}/scripts" && \
@@ -54,7 +52,7 @@ static-dist:
         )
 
 version:
-	git describe --abbrev=7 --dirty --always --tags | awk '{print "#include \"version.h\""; print "const std::string VERSION = \""$$0"\";"}' > src/version.cpp
+	grep 'version:' _config.yml | awk -F': *' '{print $2}' | awk '{print "#include \"version.h\""; print "const std::string VERSION = \""$$0"\";"}' > src/version.cpp
 
 # Clean the generated files of the main project only
 .PHONY: clean
