--- extdata/makeGroupFile.R
+++ extdata/makeGroupFile.R
@@ -1 +1,0 @@
-#!/usr/bin/env -S pixi run --manifest-path /app/pixi.toml Rscript
+#!/usr/bin/env Rscript
--- extdata/step1_fitNULLGLMM_qtl.R
+++ extdata/step1_fitNULLGLMM_qtl.R
@@ -1 +1,0 @@
-#!/usr/bin/env -S pixi run --manifest-path /app/pixi.toml Rscript
+#!/usr/bin/env Rscript
--- extdata/step2_tests_qtl.R
+++ extdata/step2_tests_qtl.R
@@ -1 +1,0 @@
-#!/usr/bin/env -S pixi run --manifest-path /app/pixi.toml Rscript
+#!/usr/bin/env Rscript
--- extdata/step3_gene_pvalue_qtl.R
+++ extdata/step3_gene_pvalue_qtl.R
@@ -1 +1,0 @@
-#!/usr/bin/env -S pixi run --manifest-path /app/pixi.toml Rscript
+#!/usr/bin/env Rscript
